Library/Homebrew/cmd/outdated.rb

```diff
@@ -38,6 +38,9 @@ def outdated_args
       switch "--greedy",
              description: "Print outdated casks with `auto_updates` or `version :latest`."
 
+      switch "--skip-unversioned",
```

Comment:
what if this was `--skip-version-latest` or `--no-version-latest`?

Reply:
I think this is a good change --skip-version-latest looks like it would be more consistent with the other options.